tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -185,7 +185,6 @@ def name(self):
             return self.root_element.text
 
         def tap_icon(self):
-            expected_name = self.name
```

Comment:
The failure you got in this latest test run is due to this, you still need to add this back.